src/otx/core/data/dataset/tile.py

```diff
@@ -105,6 +105,44 @@ def _get_item_impl(self, index: int) -> OTXDataEntity | None:
         """Get item implementation from the original dataset."""
         return self._dataset._get_item_impl(index)
 
+    def _convert_entity(self, image: np.ndarray, dataset_item: DatasetItem) -> OTXDataEntity:
+        """Convert a tile dataset item to OTXDataEntity."""
+        msg = "Method _convert_entity is not implemented."
+        raise NotImplementedError(msg)
+
+    def get_tiles(self, image: np.ndarray, item: DatasetItem) -> tuple[list[OTXDataEntity], list[dict]]:
+        """Retrieves tiles from the given image and dataset item.
+
+        Args:
+            image (np.ndarray): The input image.
+            item (DatasetItem): The dataset item.
+
+        Returns:
+            A tuple containing two lists:
+            - tile_entities (list[OTXDataEntity]): List of tile entities.
+            - tile_attrs (list[dict]): List of tile attributes.
+        """
+        tile_ds = DmDataset.from_iterable([item])
```

Comment:
I'm curious why you did apply this tile transform just to `[item]`. Is it better to apply it to `self._dataset` itself? You might also try to stack this filtering as well to exclude tiled images with no annotation https://github.com/openvinotoolkit/training_extensions/pull/2823. If it is not easy to handle this PR, can we create a backlog ticket for this?

Reply:
@vinnamkim 
1. Datumaro converts the initial image dataset, including full image annotations, into a tiled dataset with tiled annotations (tile annos are not used in evaluation). However, this transformation lacks a straightforward method to revert tiled annotations back to the original format. 

2. In the testing phase, filtering should not take place as it's not possible to know which tiles have annotations and which do not.

Reply:
Ok let me take this problem in the future.